subsys/net/ip/Kconfig

```diff
@@ -279,6 +279,22 @@ config NET_TC_NUM_PRIORITIES
 	default NUM_COOP_PRIORITIES if NET_TC_THREAD_COOPERATIVE
 	default NUM_PREEMPT_PRIORITIES if NET_TC_THREAD_PREEMPTIVE
```

Comment:
Please change the commit subject to:
`net: ip: Allow traffic class thread priority customization`
as there is no need to have multiple "net" strings there.

Reply:
updated